src/utp_stream.cpp

```diff
@@ -3408,6 +3420,22 @@ bool utp_socket_impl::incoming_packet(boost::uint8_t const* buf, int size
 	return true;
 }
 
+boost::int32_t utp_socket_impl::adjust_receive_buf_size(boost::uint32_t const& min_rtt)
```

Comment:
as far as  can tell, this function doesn't use any members of ``utp_socket_impl`` so it should be a free function (and ideally have a unit test)